cli/src/commands/migration/rollback.ts

```diff
@@ -0,0 +1,68 @@
+import { Args } from '@oclif/core';
+import { BaseCommand } from '../../base.js';
+import { getBranchDetailsWithPgRoll, isBranchPgRollEnabled } from '../../migrations/pgroll.js';
+import chalk from 'chalk';
+
+export default class MigrationRollback extends BaseCommand<typeof MigrationRollback> {
+  static description = 'Rollback an active migration';
+
+  static examples = [];
+
+  static flags = {
+    ...this.commonFlags,
+    ...this.databaseURLFlag
+  };
+
+  static args = {
+    branch: Args.string({ description: 'The branch to rollback the migration in', required: true })
+  };
+
+  static hidden = true;
+
+  async run() {
+    const { args, flags } = await this.parseCommand();
+
+    const xata = await this.getXataClient();
+    const { workspace, region, database, branch } = await this.getParsedDatabaseURLWithBranch(
```

Comment:
What happens if an invalid or non existent branch is specified?

Reply:
CLI prints: 

```bash
Error: database branch [localxb:404_branch] not found
```

As this endpoint returns 404 with data

```bash
$ curl -I -H 'Authorization: Bearer <API-KEY>' 'https://<workspace-id>.eu-west-1.staging-xata.dev/db/dbname:404_branch'
```

```bash
{"id":"60d017f9-b8ba-9211-974e-a0963da80ecd","message":"database branch [localxb:404_branch] not found"}%
```

CLI prints the error